rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -88,6 +88,8 @@
 
     public static final String PROPERTY_PRINT_SYSTEM_INFO = "system.printInfo";
 
+    public static final String PROPERTY_PREFLIGHT_CHECKS_SKIP_JAVA_VERSION_CHECK = "system.checkJavaVersion";
```

Comment:
the name of the const is still long and diff from property

Reply:
Done!